test/github/kv_test.rb

```diff
@@ -54,6 +54,43 @@ def test_set_failure
     end
   end
 
+  def test_increment_default_value
+    result = @kv.increment("foo")
+
+    assert_equal 1, result
+  end
+
+  def test_increment_large_value
+    result = @kv.increment("foo", amount: 10000)
+
+    assert_equal 10000, result
+  end
+
+  def test_increment_multiple
+    @kv.increment("foo")
+    result = @kv.increment("foo", amount: 2)
+
+    assert_equal 3, result
+  end
+
+  def test_increment_existing
+    @kv.set("foo", "1")
+
+    result = @kv.increment("foo")
+
+    assert_equal 2, result
+  end
```

Comment:
Just curious: what happens if you increment by a negative amount?

Reply:
It does go down (and can go negative). We probably don't want this. I've added some parameter checking around the amount value (as well as some tests)

Reply:
I've added back support for decrementing along with some tests.